src/plugins/autocompletion_providers/Ls.ts

```diff
@@ -0,0 +1,206 @@
+            long: "If argument is a symbolic link, list the link itself rather than the object the link references.  This option cancels the -H and -L options.",
+        },
+
+        p: {
+            short: "",
+            long: "Write a slash (`/') after each filename if that file is a directory.",
+        },
+
+        q: {
+            short: "",
+            long: "Force printing of non-graphic characters in file names as the character `?'; this is the default when output is to a terminal.",
+        },
+
+        R: {
+            short: "",
+            long: "Recursively list subdirectories encountered.",
+        },
+
+        r: {
+            short: "",
+            long: "Reverse the order of the sort to get reverse lexicographical order or the oldest entries first (or largest files last, if combined with sort by size",
+        },
+
+        S: {
+            short: "",
+            long: "Sort files by size",
+        },
+
+        s: {
+            short: "",
+            long: "Display the number of file system blocks actually used by each file, in units of 512 bytes, where partial units are rounded up to the next integer value.  If the output is to a terminal, a total sum for all the file sizes is output on a line before the listing.  The environment variable BLOCKSIZE overrides the unit size of 512 bytes.",
+        },
+
+        T: {
+            short: "",
+            long: "When used with the -l (lowercase letter ``ell'') option, display complete time information for the file, including month, day, hour, minute, second, and year.",
+        },
+
+        t: {
+            short: "",
+            long: "Sort by time modified (most recently modified first) before sorting the operands by lexicographical order.",
+        },
+
+        u: {
+            short: "",
+            long: "Use time of last access, instead of last modification of the file for sorting (-t) or long printing (-l).",
+        },
+
+        U: {
+            short: "",
+            long: "Use time of file creation, instead of last modification for sorting (-t) or long output (-l).",
+        },
+
+        v: {
+            short: "",
+            long: "Force unedited printing of non-graphic characters; this is the default when output is not to a terminal.",
+        },
+
+        W: {
+            short: "",
+            long: "Display whiteouts when scanning directories.  (-S) flag).",
+        },
+
+        w: {
+            short: "",
+            long: "Force raw printing of non-printable characters.  This is the default when output is not to a terminal.",
+        },
+
+        x: {
+            short: "",
+            long: "The same as -C, except that the multi-column output is produced with entries sorted across, rather than down, the columns.",
+        },
+
+    },
+    (option, descriptions) => mapSuggestions(shortFlag(option), suggestion => suggestion.withSynopsis(descriptions.short).withDescription(descriptions.long))
```

Comment:
Done, as well as shortening lines to fix lint.

It was pretty easy to add these.